bitbots_move_base/scripts/obstacle_publisher.py

```diff
@@ -46,23 +54,46 @@ def publish_obstacles(self, event):
                 self.ball.header,
                 [[self.ball.pose.pose.position.x, self.ball.pose.pose.position.y, self.ball.pose.pose.position.z]]))
 
-        # Publish other obstacles
-        distance = 0.05  # TODO remove after time not new message
+        # Cleanup robot obstacles
+        self.robots = list(filter(
+            lambda robot: abs((rospy.Time.now() - robot.header.stamp).to_sec()) < self.robots_storage_time,
+            self.robots))
+
+        # Enlarge robots
+        distance = 0.05
```

Comment:
Please call this something else than distance, this is very confusing. Maybe 'width' or 'size' would make more sense.